ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/IcebergPartitionEvaluator.java

```diff
@@ -33,8 +33,8 @@
 import java.util.Map;
 import java.util.Set;
 
-public class CommonPartitionEvaluator implements PartitionEvaluator {
-  private static final Logger LOG = LoggerFactory.getLogger(CommonPartitionEvaluator.class);
+public class IcebergPartitionEvaluator implements PartitionEvaluator {
```

Comment:
Why should `CommonPartitionEvaluator` be changed to `IcebergPartitionEvaluator`?

Reply:
I mainly consider that there may be other table format such as Paimon Hudi...